lib/dts-lib.robot

```diff
@@ -0,0 +1,130 @@
+    ...    option for creating HCL report works correctly.
+    Enter Shell In DTS
+    Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
+    Execute Command In Terminal    cd /
+    ${logs}=    Execute Command In Terminal
+    ...    command=/usr/bin/env DEPLOY_REPORT=false SEND_LOGS=true /usr/sbin/dasharo-hcl-report
+    ...    timeout=210s
+    Should Contain    ${logs}    Thank you
+    Should Contain    ${logs}    exited without errors
+    Should Contain    ${logs}    send completed
+
+Enter Shell In DTS
+    [Documentation]    Keyword allows to drop to Shell in the Dasharo Tools
+    ...    Suite.
+    Write Into Terminal    8
+    Login To Linux Via SSH Without Password    root    root@DasharoToolsSuite:
+
+Run EC Transition
+    [Documentation]    Keyword allows to run EC Transition procedure in the
+    ...    Dasharo Tools Suite.
+    Write Into Terminal    6
+    Read From Trminal Until    Enter an option:
+    Write Into Terminal    1
+    ${output}=    Read From Terminal Until    shut down
+    Should Contain X Times    ${output}    VERIFIED    2
+    Sleep    10s
+
+Check Power Off In DTS
+    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
+    ...    option for power off the DUT works correctly..
+    Sleep    5s
+    ${output}=    Read From Terminal
+    Length Should Be    ${output}    1
+
+Flash Firmware In DTS
+    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
+    ...    ability for flashing firmware work correctly.
+    Execute Command In Terminal
+    ...    wget -O /tmp/coreboot.rom ${FW_DOWNLOAD_LINK}
+    ${out}=    Execute Command In Terminal
+    ...    command=flashrom --ifd -i bios -p internal -w /tmp/coreboot.rom --noverify-all
+    ...    timeout=320s
+    ${verified}=    Run Keyword And Return Status
+    ...    Should Contain    ${out}    VERIFIED
+    IF    ${verified} == ${FALSE}
+        ${out}=    Execute Command In Terminal
+        ...    command=flashrom --ifd -i bios -p internal -w /tmp/coreboot.rom --noverify-all
+        ...    timeout=320s
+        Should Contain    ${out}    identical
+    END
+
+Flash EC Firmware In DTS
+    [Documentation]    Keyword allows to check if the Dasharo Tools Suite
+    ...    ability for flashing EC firmware work correctly.
+    Execute Command In Terminal
+    ...    command=wget -0 /tmp/ec.rom https://3mdeb.com/open-source-firmware/Dasharo/${EC_BINARY_LOCATION}
+    ...    timeout=320s
+    ${output}=    Execute Command In Terminal
+    ...    command=system76_ectool flash ec.rom
+    ...    timeout=320s
+    Should Contain    ${output}    Successfully programmed SPI ROM
+    Sleep    10s
+
+Check Firmware Version
+    [Documentation]    Keyword allows to check firmware version in the Dasharo
+    ...    Tools Suite Shell.
+    ${output}=    Execute Command In Terminal    dmidecode -t 0
+    Should Contain    ${output}    ${FW_VERSION}
+
+Check EC Firmware Version
```

Comment:
This does not belong to DTS. This is also specific to NovaCustom laptops only.

Reply:
Moved NVC-specific keywords to [novacustom-lib](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/44ad73e6c5e5fe00e30995617fa9463cd758d089#diff-8333448f9905ce9af4b38e9458a78e671f5dc83ff1c517aa016f907d5996329aR2)